### ISSUE TITLE
examples example

### ISSUE THREAD

**Issue by logicminds** [davetron5000/gli#281]
I am trying to implement the examples feature but it is not clear how I do this?

Can you provide some examples in the wiki?

https://github.com/davetron5000/gli/wiki/Example

**Comment by davetron5000** [comment]
Yeah, that's missing docs - I'll add.  For now: https://github.com/davetron5000/gli/pull/261
